pyfar/dsp/levels.py

```diff
@@ -0,0 +1,63 @@
+"""
+Functions used to calculate the levels of signals.
+"""
+
+import numpy as np
+import scipy.signal as sps
+
+
+def time_weighted_levels(signal, weighting: str,
```

Comment:
I dont have access to the english version. Is this the official name for that level computation? We should use the exact term from the standard in my opinion.